templates/emails/collective.expense.missing.receipt.hbs

```diff
@@ -5,16 +5,12 @@ Subject: Action Required: Submit receipt for card expense
 
 <center>
   <h1>
-    Please submit the receipt for {{fromCollective.name}} card expense.
+    Please submit the receipt for "{{expense.description}}" card expense.
   </h1>
 </center>
 <p>
-{{#if user}}
-A card attached to {{fromCollective.name}} was charged {{currency expense.amount currency=expense.currency}} and we need the responsible admin to input additional
-information.
-{{else}}
-You need to provide additional info about the card charge of {{currency expense.amount currency=expense.currency}} for {{fromCollective.name}}.
-{{/if}}
+A card attached to <a href="{{config.host.website}}/{{collective.slug}}">{{collective.name}}</a> was charged {{currency expense.amount currency=expense.currency}} for {{expense.description}} and we need the responsible admin to input additional information.
```

Comment:
What if it's a project or an event ? The slug will not be enough.

Reply:
Why have we added expense.description ?

Reply:
Expense description is NOT looking like "Team Dinner" in production so it's super misleading to take this as an example.

The current expense description looks like this: "Virtual Card charge: Zoom". Quite different ...

Reply:
The case for events/projects URLs is difficult to handle in emails, we usually don't have access to the parents in there; I voluntarily skipped it too in https://github.com/opencollective/opencollective-api/pull/7937. The redirect should do the job, we could iterate in the future by auto-loading parents from collectives before generating templates.

Reply:
@Betree : I propose a fix here, https://github.com/opencollective/opencollective-api/pull/7949 using the `getUrlPath` function. Let me know if that makes things better? 🤔 

@znarf : I've replaced the description with `fromCollective.name`.